src/Symfony/Component/HttpClient/DataCollector/HttpClientDataCollector.php

```diff
@@ -18,6 +18,8 @@
 
 /**
  * @author Jérémy Romey <jeremy@free-agent.fr>
+ *
+ * @final
```

Comment:
It's already real final, no need for annotation

Reply:
My bad indeed 👍